pyproject.toml

```
@@ -95,7 +95,7 @@ stretch = [
     "pyrender @ git+https://github.com/mmatl/pyrender.git ; sys_platform == 'linux'",
     "pyrealsense2>=2.55.1.6486 ; sys_platform != 'darwin'"
 ]
-test = ["pytest>=8.1.0", "pytest-cov>=5.0.0", "mock-serial>=0.0.1 ; sys_platform != 'win32'"]
+test = ["pytest>=8.1.0", "pytest-timeout>=2.4.0", "pytest-cov>=5.0.0", "mock-serial>=0.0.1 ; sys_platform != 'win32'"]
 hilserl = ["transformers>=4.48", "gym-hil>=0.1.5", "protobuf>=5.29.3", "grpcio==1.70.0"]
```

We should pin the version to fix the CI and ensure consistency with the codebase. Otherwise, future updates to grpcio could break the library.
```suggestion
hilserl = ["transformers>=4.48", "gym-hil>=0.1.5", "protobuf>=5.29.3", "grpcio==1.71.0"]
```